cmd/rendmp/src/lib.rs

```diff
@@ -920,6 +937,93 @@ fn rendmp_blackbox(
     Ok(())
 }
 
+fn rendmp_open_pin(
+    subargs: RendmpArgs,
+    hubris: &HubrisArchive,
+    core: &mut dyn humility::core::Core,
+    context: &mut HiffyContext,
+) -> Result<()> {
+    let (addr, dev) = check_addr(&subargs, hubris)?;
+    let op = hubris.get_idol_command("Power.rendmp_dma_read")?;
+    let regs: &'static [u16] = match dev {
+        ISL_DEV_NAME => &[
+            0x00BD, // open-pin
+            0xE925, // mask
+        ],
+        RAA_DEV_NAME => &[
+            0x00BE, 0x00BF, // open-pin
+            0xE904, 0xE905, // mask
+        ],
+        _ => panic!("wrong device ({dev})"),
```

Comment:
Tiny nit - maaaaaybe `check_addr` should return `(u8, SomeDeviceEnum)` instead of a string to avoid needing this default case?

Reply:
Good idea, done